packages/components/src/Notification/Notification.component.js

```diff
@@ -106,13 +110,17 @@ function Notification({ notification, leaveFn, ...props }) {
 	const leaveAction = event => props.onMouseOut(event, notification);
 
 	return (
-		<div // eslint-disable-line jsx-a11y/no-static-element-interactions
+		// eslint-disable-next-line jsx-a11y/click-events-have-key-events
+		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+		// eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
😢

Reply:
Yeah sorry for that, but that's not new. We can't put a button because it contains button, and there is a pin actions on notification clic